src/alpaca_eval/models_configs/Storm-7B-num-beams-10/configs.yaml

```diff
@@ -0,0 +1,14 @@
+Storm-7B-num-beams-10:
```

Comment:
this is the right name, but it seems that you generated the outputs with a different name? 

Both the leaderboard and the outputs were generated with a config named `"Storm-7B (num_beams=10)"` please change those manually or regenerate with the current config.